### PR TITLE
Add node name details in mayactl volume info command

### LINKED ISSUE #350: Add flag in mayactl volume info command (-o wide)
## Is this a BUG REPORT or FEATURE REQUEST?

 Choose one: BUG REPORT or FEATURE REQUEST
Feature Request

Add flag in mayactl volume info command to show the node name in which that volume is running.
Adding one more column of Node.

**Usage:** `mayactl volume info --volname <vol-name> -o wide` should also display the node name along with the volume details i.e wide output.

### PR DESCRIPTION
**Pull request by slalwani97**
Signed-off-by: Sumit Lalwani <sumit.lalwani97@gmail.com>

**What this PR does / why we need it**:
* Shows the volume replicas detail with the node name in which that volume is running.
* Shows volume name in mayactl volume info command.
* Adds extra column for Node in info details.

**Which issue this PR fixes**: fixes #350 

**Special notes for your reviewer**:
screenshot from 2018-06-04 12-25-39

### CODE REVIEW

File: cmd/mayactl/app/command/volume_info.go [hunk 1]
```diff
@@ -110,11 +143,20 @@ func (c *CmdVolumeInfoOptions) DisplayVolumeInfo(a *Annotations, collection clie
...
+IP             AccessMode              Status
 {{range $key, $value := .}}
 {{$value.IP}}     {{$value.AccessMode}}                       {{$value.Status}}
 {{end}}
 ===================================================
+`
+
+		replicaTemplateWithNodeName = `
+================= Replica Details ==================================
+IP            AccessMode               Status           Node
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/volume_info.go]: @slalwani97  columns are not aligned properly as i can see in the output ?

**Comment by slalwani97** [comment]
screenshot from 2018-06-01 18-23-49

fixed the alignment and removed flag

**Comment by kmova** [comment]
@slalwani97 This PR is coming nicely. The alignment could still be an issue. Can you try to run this command on a volume that is generated by PVC?

**Comment by slalwani97** [comment]
@kmova Sir, yes alignment was an issue because volume names can be of any length. Now I have used tabwriter with template to align properly.

**Comment by slalwani97** [comment]
screenshot from 2018-06-04 01-43-06

**Comment by NisantaSahoo** [comment]
******Star PR **********

**Pull request merged**